Sources/Misc/Deprecations.swift

```diff
@@ -389,3 +389,26 @@ extension CustomerInfo {
     }
 
 }
+
+public extension Configuration.Builder {
+    ///
+    /// Set `usesStoreKit2IfAvailable`. If `true`, the SDK will use StoreKit 2 APIs internally. If disabled, it will use StoreKit 1 APIs instead.
+    /// - Parameter usesStoreKit2IfAvailable: enable StoreKit 2 on devices that support it.
+    /// Defaults to  `false`.
+    /// - Important: This configuration flag has been deprecated, and will be replaced by automatic remote configuration in the future.
+    /// However, apps using it should work correctly.
+    ///
+    @available(*, deprecated, message: """
+    RevenueCat currently uses StoreKit 1 for purchases, as its stability in production scenarios has overall
+    proved more better than StoreKit 2.
```

Comment:
I would change this:

> proved more better

to

> proved more performant

Reply:
lol "more better" 
I just kept updating the copy until I totally messed it up